src/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -16,47 +16,44 @@
  */
 
 // Import helpers from zap core
-const zapPath      = '../../../../../third_party/zap/repo/src-electron/';
```

Comment:
The random reindenting here is not great.....

Reply:
This indent got fixed in the second commit when i ran the restyler. I this its my  vscode formating that screwed something there the first time when i did save. Should be ok now. 

And for the failing test I will take a look, Most of it is generated code so I am suprise it fails